Win11ThemeTest/ComboBoxTest.cs

```diff
@@ -0,0 +1,130 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.Core.Tools;
+using FlaUI.UIA3;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Win11ThemeTest
+{
+    public class ComboBoxTest
+    {
+        Application app;
+        Window MainWindow;
+        Button cmbBoxButton;
+        Window comboWindow;
+        ComboBox comboBox;
+        ComboBox comboBoxBind;
+        ComboBox comboBoxBind2;
+        UIA3Automation automation = new UIA3Automation();
```

Comment:
prefer initializing this inside constructor

Reply:
Added inside constructor